controls/tests.py

```diff
@@ -280,9 +276,8 @@ def test_component_import_oscal_json(self):
 
         self.click_element('input#import_component_submit')
 
-        var_sleep(3) # Wait for OSCAL to be imported
+        element_count_after_import = wait_for_sleep_after(lambda: Element.objects.filter(element_type="system_element").count())
```

Comment:
Here, we want to wait for sleep after the previous command (`click_element`), right? Here, I think the wait is being called on the wrong function.

Reply:
So the wait_for is applied to allow whatever happened directly after a var_sleep to occur without issue. So if it needs more than 3 or less than 3 this allows in this case element_count_after_import  to still be created.

Reply:
I see. I think the phrasing of the function threw me off. It initially had me thinking "Wait for sleep after doing the param function" which is backwards. In that case, I think "wait before" would be more accurate, correct? What about something like "wait_then_execute"? With the combo of the function name as the parameter, it might read intuitively, like: `wait_then_execute(some_function)`